src/lib/config/CommonContentClientConfig.ts

```diff
@@ -22,7 +17,7 @@ export interface ContentClientConfig {
   stagingEnvironment?: string;
 
   /**
-   * If set, the SDK will request content using the locale settings provided.
+   * If set, the SDK will request content using the locale settings provided.rm -fr
```

Comment:
`rm -fr` typo

Reply:
Must have stolen focus while I was cleaning out the node_modules during tests.